### PR TITLE
5582 - Setting the duration property on an event leaves it undefined …

### PR DESCRIPTION
**Pull request by pvirlan**
…in extendedProps

https://github.com/fullcalendar/fullcalendar/issues/5582

### CODE REVIEW

**Comment by arshaw** [comment]
When i wrote my ambiguous instructions...

> Could you also do the same exact thing for the "simple" recurring event properties (daysOfWeek/startTime/endTime/etc)

What I actually meant is that we should make references to all the simple recurring event properties (daysOfWeek/startTime/endTime/etc) **from within the Event Parsing article**.

The end-goal is to look at the Event Parsing article and see all the possible event properties all on that one page. This goes beyond the scope of the original issue (solely about the `duration` property) and attempts to preempt the problem of someone not knowing why `daysOfWeek` isn't making it to `extendedProps` too.

Does that make sense?

File: _docs-v5/upgrading-from-v4.md [hunk 1]
```diff
@@ -1529,6 +1530,40 @@ In list view, the "No events to display" message.
...
+<table>
+  <tr>
+    <td>
+      <ul class='diff-list'>
+        <li class='diff-removed'>duration</li>
+      </ul>
+    </td>
+<td markdown='1'>
+
+In v4, the events using duration property were added to event.extendedProps and was accesible from there, however in v5 this property is now an Event class property and will not be accesible from event.extendedProps anymore, a workaround would be to provide an alternative name like in [this example](https://codepen.io/LiamBull/pen/mdVpwme)
```
Comment (arshaw) [inline _docs-v5/upgrading-from-v4.md:1544]: could you remove the codepen link? i like to keep all runnable examples within the codebase. and a runnable demo is overkill in this scenario

Comment (pvirlan) [reply _docs-v5/upgrading-from-v4.md]: makes sense, done

**Comment by pvirlan** [comment]
> When i wrote my ambiguous instructions...
> 
> > Could you also do the same exact thing for the "simple" recurring event properties (daysOfWeek/startTime/endTime/etc)
> 
> What I actually meant is that we should make references to all the simple recurring event properties (daysOfWeek/startTime/endTime/etc) **from within the Event Parsing article**.
> 
> The end-goal is to look at the Event Parsing article and see all the possible event properties all on that one page. This goes beyond the scope of the original issue (solely about the `duration` property) and attempts to preempt the problem of someone not knowing why `daysOfWeek` isn't making it to `extendedProps` too.
> 
> Does that make sense?

yes, do you want me to add a comment as well about these params that they're not going to make it to extendedProps hash?

**Comment by arshaw** [comment]
looks good 👍 

```
<div style='white-space:nowrap'>rrule</div> duration 
```

to this:

```
<div style='white-space:nowrap'>rrule</div>
<div style='white-space:nowrap'>duration</div> 
```

because it's more readable, but don't worry, I can do that myself. 

i'll merge and release

**Comment by pvirlan** [comment]
> looks good 👍
> 
> i would change this:
> 
> ```
> <div style='white-space:nowrap'>rrule</div> duration 
> ```
> 
> to this:
> 
> ```
> <div style='white-space:nowrap'>rrule</div>
> <div style='white-space:nowrap'>duration</div> 
> ```
> 
> because it's more readable, but don't worry, I can do that myself.
> 
> i'll merge and release

yeah, looks much nicer like that, thx
also I've made the mistake to create the pull request from master branch from my forked repo on this one :(

**Pull request merged**